models/weight_recorder.h

```diff
@@ -34,10 +34,12 @@
 #include "nest_types.h"
 #include "recording_device.h"
 
-/* BeginUserDocs: device, recorder
+/* BeginUserDocs: device, recorder, synapse
```

Comment:
As discussed, I reverted various doc content changes, as this will be fixed by other PRs and does not concern the extractor per se, so this would also not go into this PR.